recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: distributed-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/d/distributed/distributed-{{ version }}.tar.gz
-  sha256: fc9c3a7b27bd2f76aead3016ff147d80396dfa147e1be92a732d836a99817254
+  sha256: fa37da099f589b673006b95b2bdee91930c5609d1b6d25300a4518df2c4e7be1
 
 build:
   number: 0
```

Comment:
Might need to merge with `master` and reset to `0` as it is is now `1` in `master`.